packages/stacked/lib/src/base_view_models.dart

```diff
@@ -238,10 +244,15 @@ abstract class MultipleFutureViewModel extends _MultiDataSourceViewModel {
   void onError({String key, error}) {}
 
   void onData(String key) {}
+
+  initialise() {
```

Comment:
Instead of having a function that calls the runFutures function. Change runFutures to be the initialise function from the Initialisable interface.

Reply:
Renamed the function.